ApplicationLibCode/ProjectDataModel/VerticalFlowPerformance/RimVfpPlotCollection.cpp

```diff
@@ -32,33 +32,10 @@ RimVfpPlotCollection::RimVfpPlotCollection()
 {
     CAF_PDM_InitObject( "VFP Plots", ":/VfpPlotCollection.svg" );
 
-    CAF_PDM_InitFieldNoDefault( &m_vfpPlots, "VfpPlots", "Vertical Flow Performance Plots" );
+    CAF_PDM_InitFieldNoDefault( &m_vfpPlots_deprecated, "VfpPlots", "Vertical Flow Performance Plots" );
     CAF_PDM_InitFieldNoDefault( &m_customVfpPlots, "CustomVfpPlots", "Vertical Flow Performance Plots" );
 }
 
```

Comment:
Might be hard, but could be better to rename CustomVfpPlot to VfpPlot and handle the conversion to initAfterRead?

It was custom before, but now it is the only VfpPlot available.

Reply:
This is difficult now. Implement the suggested change when the obsoleted VFP plot is removed.